.ko.yaml

```
@@ -0,0 +1,74 @@
+defaultBaseImage: ghcr.io/edgelesssys/apko-alpine-base:latest
+
+baseImageOverrides:
+  github.com/edgelesssys/constellation/operators/constellation-node-operator/v2: ghcr.io/edgelesssys/apko-alpine-base-user-65532
+  github.com/edgelesssys/constellation/v2/hack/qemu-metadata-api: ghcr.io/edgelesssys/apko-alpine-qemu-metadata-api
+
+builds:
+- id: kms
+  dir: .
+  main: ./kms/cmd
+  env:
+  - CGO_ENABLED=0
```

```suggestion
  - CGO_ENABLED=0
  - GOOS=linux
  - GOARCH=amd64
```
Is there a reason why kms and joinservice lack GOOS and GOARCH? Same for missing `-a `flag.

---

No there is no reason for that. I think your idea is good to make it more explicit.